#ifndef SPDK_CONFIG_H
#define SPDK_CONFIG_H
#define SPDK_CONFIG_APPS 1
#define SPDK_CONFIG_ARCH native
#undef SPDK_CONFIG_ASAN
#undef SPDK_CONFIG_AVAHI
#undef SPDK_CONFIG_CET
#define SPDK_CONFIG_COPY_FILE_RANGE 1
#undef SPDK_CONFIG_COVERAGE
#define SPDK_CONFIG_CROSS_PREFIX 
#undef SPDK_CONFIG_CRYPTO
#undef SPDK_CONFIG_CUSTOMOCF
#undef SPDK_CONFIG_DAOS
#define SPDK_CONFIG_DAOS_DIR 
#undef SPDK_CONFIG_DEBUG
#define SPDK_CONFIG_DPCP_DIR 
#undef SPDK_CONFIG_DPDK_COMPRESSDEV
#define SPDK_CONFIG_DPDK_DIR /home/evgenybud/projects/repos/nebius/storage/github.mellanox.spdk/dpdk/build
#define SPDK_CONFIG_DPDK_INC_DIR 
#define SPDK_CONFIG_DPDK_LIB_DIR 
#undef SPDK_CONFIG_DPDK_PKG_CONFIG
#undef SPDK_CONFIG_DPDK_UADK
#define SPDK_CONFIG_ENV /home/evgenybud/projects/repos/nebius/storage/github.mellanox.spdk/lib/env_dpdk
#define SPDK_CONFIG_EXAMPLES 1
#undef SPDK_CONFIG_FC
#define SPDK_CONFIG_FC_PATH 
#undef SPDK_CONFIG_FIO_PLUGIN
#define SPDK_CONFIG_FIO_SOURCE_DIR /usr/src/fio
#define SPDK_CONFIG_FSDEV 1
#undef SPDK_CONFIG_FUSE
#undef SPDK_CONFIG_FUZZER
#define SPDK_CONFIG_FUZZER_LIB 
#undef SPDK_CONFIG_GOLANG
#define SPDK_CONFIG_HAVE_ARC4RANDOM 1
#define SPDK_CONFIG_HAVE_EVP_MAC 1
#define SPDK_CONFIG_HAVE_EXECINFO_H 1
#define SPDK_CONFIG_HAVE_FANOTIFY 1
#undef SPDK_CONFIG_HAVE_KEYUTILS
#undef SPDK_CONFIG_HAVE_LIBARCHIVE
#undef SPDK_CONFIG_HAVE_LIBBSD
#undef SPDK_CONFIG_HAVE_LZ4
#define SPDK_CONFIG_HAVE_STRUCT_STAT_ST_ATIM 1
#undef SPDK_CONFIG_HAVE_STRUCT_STAT_ST_ATIMESPEC
#define SPDK_CONFIG_HAVE_UUID_GENERATE_SHA1 1
#undef SPDK_CONFIG_IDXD
#undef SPDK_CONFIG_IDXD_KERNEL
#undef SPDK_CONFIG_IPSEC_MB
#define SPDK_CONFIG_IPSEC_MB_DIR 
#undef SPDK_CONFIG_ISAL
#undef SPDK_CONFIG_ISAL_CRYPTO
#undef SPDK_CONFIG_ISCSI_INITIATOR
#define SPDK_CONFIG_LIBDIR 
#undef SPDK_CONFIG_LTO
#define SPDK_CONFIG_MAX_LCORES 128
#define SPDK_CONFIG_MAX_NUMA_NODES 1
#define SPDK_CONFIG_NVME_CUSE 1
#undef SPDK_CONFIG_OCF
#define SPDK_CONFIG_OCF_PATH 
#define SPDK_CONFIG_OPENSSL_PATH 
#undef SPDK_CONFIG_PGO_CAPTURE
#define SPDK_CONFIG_PGO_DIR 
#undef SPDK_CONFIG_PGO_USE
#define SPDK_CONFIG_PREFIX /usr/local
#undef SPDK_CONFIG_RAID5F
#undef SPDK_CONFIG_RBD
#undef SPDK_CONFIG_RDMA
#define SPDK_CONFIG_RDMA_PROV verbs
#undef SPDK_CONFIG_RDMA_SEND_WITH_INVAL
#undef SPDK_CONFIG_RDMA_SET_ACK_TIMEOUT
#undef SPDK_CONFIG_RDMA_SET_TOS
#undef SPDK_CONFIG_SHARED
#undef SPDK_CONFIG_SMA
#undef SPDK_CONFIG_STATIC_XLIO
#define SPDK_CONFIG_TESTS 1
#undef SPDK_CONFIG_TSAN
#undef SPDK_CONFIG_UBLK
#undef SPDK_CONFIG_UBSAN
#undef SPDK_CONFIG_UNIT_TESTS
#define SPDK_CONFIG_URING 1
#define SPDK_CONFIG_URING_PATH 
#define SPDK_CONFIG_URING_ZNS 1
#undef SPDK_CONFIG_USDT
#undef SPDK_CONFIG_VBDEV_COMPRESS
#undef SPDK_CONFIG_VBDEV_COMPRESS_MLX5
#define SPDK_CONFIG_VFIO_USER 1
#define SPDK_CONFIG_VFIO_USER_DIR 
#define SPDK_CONFIG_VHOST 1
#define SPDK_CONFIG_VIRTIO 1
#undef SPDK_CONFIG_VTUNE
#define SPDK_CONFIG_VTUNE_DIR 
#undef SPDK_CONFIG_WERROR
#define SPDK_CONFIG_WPDK_DIR 
#undef SPDK_CONFIG_XLIO
#define SPDK_CONFIG_XLIO_DIR 
#undef SPDK_CONFIG_XNVME
#endif /* SPDK_CONFIG_H */
